v2/datastream-to-spanner/src/test/resources/DataStreamToSpannerShardedMigrationIT/spanner-schema.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE IF NOT EXISTS Users (
+    id INT64 NOT NULL,
+    name STRING(200),
+    age_spanner INT64,
+    migration_shard_id STRING(50),
+) PRIMARY KEY (migration_shard_id, id);
+
+CREATE TABLE IF NOT EXISTS Movie (
+    id1 INT64 NOT NULL,
+    id2 INT64 NOT NULL,
+    name STRING(200),
+    actor INT64,
+    migration_shard_id STRING(50),
+) PRIMARY KEY (id2, id1, migration_shard_id);
```

Comment:
Lets cover more breadth in the data types.

Reply:
We have another test case to cover all the datatypes. In these test cases, I covered only Sharding specific logic .i.e., migration_shard_id column being populated correctly and multiple physical+logical shard migration happening simultaneously.

Reply:
Can you point me to that test case please?